cmd/limactl/start.go

```diff
@@ -529,3 +529,55 @@ func startBashComplete(cmd *cobra.Command, _ []string, _ string) ([]string, cobr
 	compTmpl, _ := bashCompleteTemplateNames(cmd)
 	return append(compInst, compTmpl...), cobra.ShellCompDirectiveDefault
 }
+
+// startInteractive starts the instance after a confirmation prompt.
+//
+// If newInst is set to true, this function creates the instance too.
+//
+// If mandatory is set to true, and the answer to the confirmation prompt is No,
+// the function returns an error that contains an instruction to start the instance manually.
+func startInteractive(cmd *cobra.Command, instName string, newInst, mandatory bool) (bool, error) {
```

Comment:
It is not clear what the `bool` return value represents. It also seems to be completely unused; all 3 call-sites are `_, err = startInteractive(…)`.